modules/ROOT/pages/import-api-specification-exchange.adoc

```diff
@@ -55,8 +55,62 @@ Manage the API specifications linked to Exchange in your project from the API sp
 [NOTE]
 If you want Studio to list your API when searching in Exchange, you must mark your API specification published as an asset as *stable*.
 
+== Implement AsyncAPI Specifications
+
+Studio supports the implementation of AsyncAPI 2.6 specs. When creating an implementation project, Studio imports and scaffolds an AsyncAPI spec hosted on Exchange into an API interface that you can implement.
+
+For supported protocols, see xref:apikit::apikit-asyncapi-module-reference.adoc#supported-message-brokers[Supported Message Brokers] in APIkit for AsyncAPI Module Reference.
+
+=== Access AsyncAPI Implementation Features
+
+. Open Anypoint Code Builder landing page on your Anypoint Platform cloud host (https://anypoint.mulesoft.com/codebuilder/[US] or https://eu1.anypoint.mulesoft.com/codebuilder/[EU]). 
++
+Sign-in is required. The *Terms & Conditions* button is available to the organization administrator only.
+. If the message *AsyncAPI Beta is now available in Anypoint Code Builder and Anypoint Studio* appears on the page, ask your organization administrator to join the Beta program by:
+.. Clicking *Join Beta* from the Anypoint Code Builder landing page.
```

Comment:
no period at the end of this because it is a sentence fragment, not a complete sentence

Reply:
It might help to add the screenshot you see in the ACB docs for this.

Reply:
Done with the period! ;) 

No need to add the screenshot from my point of view.